platforms/lv_conf.h

```diff
@@ -12,23 +12,22 @@
  */
 
 /* clang-format off */
-#ifndef QUANTUM_PAINTER_LVGL_USE_CUSTOM_CONF
+#if 1 /*Set it to "1" to enable content*/
 
 #ifndef LV_CONF_H
 #define LV_CONF_H
 
 #include <stdint.h>
-#include "qp_lvgl.h"
 
 /*====================
    COLOR SETTINGS
  *====================*/
 
 /*Color depth: 1 (1 byte per pixel), 8 (RGB332), 16 (RGB565), 32 (ARGB8888)*/
-#define LV_COLOR_DEPTH QUANTUM_PAINTER_LVGL_COLOR_DEPTH
+#define LV_COLOR_DEPTH 16
```

Comment:
Do all the displays that QP supports are RGB565? Worst case scenario is that users will need to use a custom `lv_conf.h` to overwrite if using a display with a different color depth.

Reply:
Right now, yes, they're all 16bit.

The configuration override file allows for overriding the defaults much like the ChibiOS config override files when placed in the keyboard directory. The override file allows overriding a handful of entries rather than supplying the whole file, simplifying maintenance. And if people don't need to change a setting... no override needed.